packages/zoe/test/swingsetTests/zoe/vat-bob.js

```diff
@@ -87,12 +87,8 @@ const build = async (E, log, zoe, issuers, payments, installations, timer) => {
       const { extent: optionExtent } = await E(inviteIssuer).getAmountOf(
```

Comment:
Since we always want `optionExtent[0]`, would it make sense to do

``` js
const { extent: [optionExtent] } = await E(inviteIssuer).getAmountOf(
 ...
```

Reply:
I think we do that in some of our tests (or we did, before we changed that code for the helper.) I think it's a matter of preference, but I think people miss that the extent is an array, so I kind of like to make that explicit. Your version is neater, though.

Reply:
I think I'm going to keep it with the `[0]` index because calling the first element of the extent an extent doesn't seem quite right.